databrowser/src/domain/cellComponents/components/cells/editedDateCell/EditedDateCell.vue

```diff
@@ -22,12 +22,44 @@ const props = withDefaults(
 
 const { date, format } = toRefs(props);
 
+const nth = function (d) {
+  if (d > 3 && d < 21) return 'th';
+  switch (d % 10) {
+    case 1:
+      return 'st';
+    case 2:
+      return 'nd';
+    case 3:
+      return 'rd';
+    default:
+      return 'th';
+  }
+};
+
 const formattedDate = computed(() => {
   if (format == null) {
     return date;
   }
   if (date != null) {
-    return formatFn(Date.parse(date.value), format.value);
+    const fullDate = new Date(Date.parse(date.value));
+    const day = fullDate.getDate();
+    const month = [
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+    ][fullDate.getMonth()];
+    const year = fullDate.getFullYear();
+
+    return day + nth(day) + ' ' + month + ' ' + year;
```

Comment:
I think the library date-fns should be capable of formatting the date according to our needs, please check https://date-fns.org/v2.29.2/docs/format. This would simplify the implementation

Reply:
I'm sorry for some reason I didn't find this documentation earlier. I will eliminate this vue component as it is useless and also restore the editDateCell. I will just change the format in the singular config files to:  do MMMM yyyy where is needed.
Thank you for the info.